scripts/lib/mkcloud-onhost.sh

```diff
@@ -121,7 +121,7 @@ function onhost_cacheclouddata
         for a in $architectures; do
             local suffix
             # Mirror SLES/HA/SOC update + pool repos
-            for suffix in Pool Updates; do
+            for suffix in Pool Updates Updates-test; do
```

Comment:
@matelakat @dirkmueller the problem is that if $hacloud is set, it will trying to mount the HA-Updates-Test repo. See

https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crowbarsetup.sh#L319

Perhaps $hacloud also implies $want_test_updates? So I make that change as well?